frontend/app/src/store/defi/actions.ts

```diff
@@ -409,6 +409,7 @@ export const actions: ActionTree<DefiState, RotkehlchenState> = {
       dispatch('fetchDSRBalances', refresh),
       dispatch('fetchMakerDAOVaults', refresh),
       dispatch('fetchCompoundBalances', refresh),
+      dispatch('fetchPickleBalances', refresh),
```

Comment:
Did we implement this action in another PR? if not this should log something about a missing action when called.